src/node/mod.rs

```diff
@@ -0,0 +1,384 @@
+//! Module for abstractions on drm device nodes.
+
+pub mod constants;
+
+use std::error::Error;
+use std::fmt::Debug;
+use std::fmt::{self, Display, Formatter};
```

Comment:
Merge? I personally prefer qualifying `fmt::` everywhere to not import `::{self}`.